packages/wallets/src/components/AccountsList/AccountsList.tsx

```diff
@@ -23,9 +24,11 @@ const AccountsList: FC<TSubscribedBalance> = ({ balance }) => {
                 <WalletsPrimaryTabPanels>
                     <WalletsPrimaryTabPanel>
                         <CFDPlatformsList />
+                        <Divider color='#e6e9e9' />
```

Comment:
@lubega-deriv does @deriv-com/ui provide a way to handle theme and color variables?

Reply:
@nijil-deriv I believe currently they don't and that's how it is done in the component itself:
https://github.com/deriv-com/ui/blob/main/src/components/Divider/index.tsx


<img width="1055" alt="image" src="https://github.com/binary-com/deriv-app/assets/142860499/3a02867a-da91-4a9d-aa94-956e6c85c4a1">

Reply:
Can you use the CSS variables instead of the direct hex codes?

Reply:
Updated